build.gradle

```diff
@@ -170,7 +171,12 @@ dependencies {
     implementation 'com.tobiasdiez:easybind:2.2.1-SNAPSHOT'
     implementation 'org.fxmisc.flowless:flowless:0.7.0'
     implementation 'org.fxmisc.richtext:richtextfx:0.11.0'
-    implementation 'com.jfoenix:jfoenix:9.0.10'
+    implementation (group: 'com.dlsc.gemsfx', name: 'gemsfx', version: '1.72.0') {
+        exclude module: 'javax.inject' // Split package, use only jakarta.inject
+        exclude group: 'com.dlsc.unitfx'
+        exclude group: 'org.apache.logging.log4j'
```

Comment:
We should file a PR to gemsfx to replace log4j by slf4j and that logging implementations are shipped for testing only.

Reply:
This may be an artifact from playing around with the dependencies. @Siedlerchr tried something here.

Reply:
Yep, unitfx was some tests and log4j seemed to be inside the classpath/modulepath as well.